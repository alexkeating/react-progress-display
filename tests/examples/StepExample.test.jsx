import React from 'react';
import renderer from 'react-test-renderer';
import StepExample from '../../examples/src/StepProgressBarExample';

test('StepExample snapshot', () => {
  const component = renderer.create(
    <StepExample />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

