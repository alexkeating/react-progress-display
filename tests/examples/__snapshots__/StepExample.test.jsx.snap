// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepExample snapshot 1`] = `
<div>
  <div
    className="progress-bar"
  >
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
    <li
      className="progress-bar-disabled"
    >
      <span />
    </li>
  </div>
  <button
    onClick={[Function]}
  >
    Forward
  </button>
  <button
    onClick={[Function]}
  >
    Backward
  </button>
</div>
`;
